lib/awful/tag.lua

```diff
@@ -873,6 +874,14 @@ function tag.mt:__call(...)
     return tag.new(...)
 end
 
+-- Extend the luaobject
+object.properties(capi.tag, {
+    getter_class    = tag,
+    setter_class    = tag,
+    getter_fallback = tag.getproperty,
+    setter          = tag.setproperty,
```

Comment:
Perhaps a comment explaining why the getter is fallback, but the setter is not? I looked at the code and I think I understand the difference between the two, but I'm not sure why it's necessary to do it like this here.